### PR TITLE
fixed errors in commands names 

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/32128

## Description
Fix to integrations that failed on the new xsoar linter validation of commands implementation:
for those integrations some commands were found in the yml file and were not in the python file because of errors in the names of the commands

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CloudShare/Integrations/CloudShare/CloudShare.py [hunk 1]
```diff
@@ -1170,6 +1170,7 @@ def main() -> None:
             'cloudshare-get-class': get_class_command,
             'cloudshare-delete-class': delete_class_command,
             'cloudshare-delete-class-environemtns': delete_class_environments_command,
+            'cloudshare-delete-class-environemnts': delete_class_environments_command,
```
Comment (ShahafBenYakir) [inline Packs/CloudShare/Integrations/CloudShare/CloudShare.py:1173]:
```suggestion
            'cloudshare-delete-class-environemtns': delete_class_environments_command,    # This is here for maintaining BC
            'cloudshare-delete-class-environemnts': delete_class_environments_command,
```

File: Packs/Infinipoint/Integrations/Infinipoint/Infinipoint.yml [hunk 2]
```diff
@@ -841,7 +841,7 @@ script:
     - contextPath: Infinipoint.Scripts.execute.name
       description: Query name
       type: String
-  - name: infinipoint-get-non-compliance
+  - name: infinipoint-get-events
```
Comment (ShahafBenYakir) [inline Packs/Infinipoint/Integrations/Infinipoint/Infinipoint.yml:844]: How do we know this is the related command? And why not change it in the python file? this way people will not have to change playbooks and stuff.

File: Packs/WootCloud/Integrations/WootCloud/WootCloud.yml [hunk 3]
```diff
@@ -792,7 +792,7 @@ script:
       - contextPath: WootCloud.PacketAlert.references
         description: 'references'
         type: String
-    - name: wootcloud-fetch-bluetooth-alert
+    - name: wootcloud-fetch-bt-alert
```
Comment (ShahafBenYakir) [inline Packs/WootCloud/Integrations/WootCloud/WootCloud.yml:795]: Why not change the python?

**Review by ShahafBenYakir — COMMENTED** [review]
Let's make sure that we test these commands in the test pb as well

**Review by ShahafBenYakir — APPROVED** [review]

**Review by roysagi — COMMENTED** [review]
@moishce - please refrain from using `Maintenance and stability enhancements.` when possible - in these cases we should indicate which command names changed or were not handled correctly so the users know they are now useable

**Comment by ShahafBenYakir** [comment]
@roysagi Can you re-review?

**Comment by ShahafBenYakir** [comment]
Braking changes made ion purpose - force merging

**Pull request merged**